widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -9325,3 +9393,64 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
     text-overflow: ellipsis;
   }
 }
+
+/* endregion Main Toolbar */
+
+.schedulesPanel .toolbar {
+  background: none;
+  border: 0;
+  margin-bottom: 5px;
+}
+
+.schedulesPanel .pentaho-table th {
+  font-size: 12px;
+  border-spacing: 0;
+  color: #FFF;
+  padding: 8px 10px 10px 10px;
+  font-weight: normal;
+}
```

Comment:
These two rules belong to the Schedules perspective, which is loaded inside Mantle. Should be moved back to `mantleRuby.css`.

Reply:
Moved back.